### PR TITLE
pass FLUTTER_TARGET_PLATFORM flag to cmake.

### PR DESCRIPTION
**Pull request by cloudwebrtc**
fix https://github.com/sony/flutter-elinux/issues/149

### CODE REVIEW

**Comment by cloudwebrtc** [comment]
@HidenoriMatsubayashi okay, updated.

**Review by HidenoriMatsubayashi — COMMENTED** [review]
Oh, sorry, I forgot to submit my comment.

File: lib/elinux_build_target.dart [hunk 1]
```diff
@@ -365,6 +365,7 @@ class NativeBundle {
         'cmake',
         '-DCMAKE_BUILD_TYPE=$cmakeBuildType',
         '-DFLUTTER_TARGET_BACKEND_TYPE=${buildInfo.targetBackendType}',
+        '-DFLUTTER_TARGET_PLATFORM=linux-${buildInfo.targetArch}',
```
[the remark this answers is not in the document]
Comment (cloudwebrtc) [reply lib/elinux_build_target.dart:368]: yeah, that makes sense

**Review by HidenoriMatsubayashi — APPROVED** [review]
Thanks.

**Pull request merged**